net/pptpd/files/pptpd.init

```diff
@@ -12,7 +12,8 @@ OPTIONS_PPTP=/var/etc/options.pptpd
 validate_login_section() {
 	uci_validate_section pptpd login "${1}" \
 		'username:string' \
-		'password:string'
+		'password:string' \
+		'remoteip:string'
```

Comment:
the validation spec should better be `remoteip:or(list(ipaddr),"*"):"*"`

Reply:
It uses the same validation as 'localip' in the pptpd section, for coherence should we change that too?
```
validate_pptpd_section() {
	uci_validate_section pptpd service "${1}" \
		'enabled:uinteger' \
		'localip:string' \
		'remoteip:string' \
		'mppe:list(string):required no40 no56 stateless' \
		'logwtmp:uinteger'
}
```

Reply:
Yeah, but better do it in a separate commit ;)

Reply:
Thanks! I will do it and then test it when I have some spare time! Then I submit it too!
Best Regards and thanks again for the help!